provisionerd/proto/version.go

```diff
@@ -25,9 +25,12 @@ import "github.com/coder/coder/v2/apiversion"
 //   - Add previous parameter values to 'WorkspaceBuild' jobs. Provisioner passes
 //     the previous values for the `terraform apply` to enforce monotonicity
 //     in the terraform provider.
+//
+// API v1.7:
+//   - Add `group` field to `App`
 const (
 	CurrentMajor = 1
-	CurrentMinor = 6
+	CurrentMinor = 7
```

Comment:
AFAICT provisionerd proto version 1.6 has not yet been released (see: https://github.com/coder/coder/blob/release/2.22/provisionerd/proto/version.go#L21-L24). 

Do we want to roll it into that version?